packages/dds/tree/src/test/utils.ts

```diff
@@ -148,9 +147,14 @@ import type { Client } from "@fluid-private/test-dds-utils";
 import { JsonUnion, cursorToJsonObject, singleJsonCursor } from "./json/index.js";
 // eslint-disable-next-line import/no-internal-modules
 import type { TreeSimpleContent } from "./feature-libraries/flex-tree/utils.js";
+import type {
+	RevertibleAlpha,
+	RevertibleFactoryType,
+	RevertibleType,
+	// eslint-disable-next-line import/no-internal-modules
+} from "../core/revertible.js";
```

Comment:
@Josmithr 

I've decided to refactor the build-undo/redo-stack functions a bit since `createTestUndoRedoStacks` & `createClonableUndoRedoStacks` had almost identical code. 


I __think__ this forced me to break the `no-internal-modules` rule, since `RevertibleAlpha` (and also `RevertibleAlphatFactory`) is `alpha` for now. 

Do you think this is okay?

Reply:
Why is any of that necessary? Can't you just use `createClonableUndoRedoStacks` everywhere that `createTestUndoRedoStacks` was used (and, in that case, don't ever bother renaming it)? Tests that don't care about cloning can still receive Revertibles that are clonable - they'll just ignore the `clone()` method.

I think it is likely that _all_ of our internal test code should simply be updated to use `RevertibleAlpha`. No need for any bifurcation.

Reply:
This totally makes sense now! Thanks for pointing ou the mistake. Changes applied 😄 .